stable/prefer-node-descheduler/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for prefer-node-descheduler.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount -- Number of replicas to run.
+replicaCount: 1
+
+
+image:
+  # image.repository -- The docker repository and image to be used.
+  repository: nginx
```

Comment:
I think this is a mistake?

Reply:
Yes and no. I don't have a docker image ready yet and I'm still discussing where the image will be stored (GCR, Dockerhub or any other). I'll update it as soon as I have it.